src/content/whats-new/2022/03/whats-new-03-28-futurestack-22.md

```diff
@@ -0,0 +1,41 @@
+---
+title: 'FutureStack is back—in-person at the Cosmopolitan of Las Vegas' 
+summary: 'Join us May 17-19 for inspiring keynotes, hands-on labs, technical breakouts, 
+and amazing entertainment, Vegas style.' 
+releaseDate: '2022-03-28' 
+learnMoreLink: 'https://www.futurestack.com/anqQXE?RefId=WHATSNEW' 
+---
+
+![FutureStack22](./images/FutureStack.jpg "FutureStack22")
+
+
+**Join us for FutureStack May 17-19 at the Cosmopolitan of Las Vegas**
+
+Reconnect face-to-face and node-to-node—and find out what your most innovative peers are doing.
+
+[Register Now](https://www.futurestack.com/anqQXE?RefId=WHATSNEW)
+
+Explore observability with inspiring keynotes, peer-led deep dives, and exclusive technical breakouts you won’t find online.
+
+**FutureStack is back, bigger and bolder than ever.** 
+It’s time to log off from virtual events, dust off your suitcase, and join your peers to rub elbows and swap cheat codes in a way that’s only possible in live, in-person interactive sessions. 
+
+**Reasons all the data points to Las Vegas.**
+* **1:1 expert guidance** for a deep dive on observability
+* **An inside look** at the newest New Relic innovations
+* **Blockbuster announcements** for new products and more
+* **Networking events** and exclusive entertainment, Vegas style
+
+**Look who’s sharing their observability insights.**
+Our peer-led sessions feature industry leaders, including:
+* **JJ Asghar** - IBM
+* **Carl Chesser** - Cerner 
+* **Sheldon Lessard** - Maricopa County Clerk of the Superior Court
+
+**Drop in and code on at the 24-hour FutureHack.**
```

Comment:
@Alanjbaptista is this a typo? 'code on at'?

Reply:
No it's now. It's the author trying to be cute. Maybe if we say ...."code-on at Future Stack"